pkg/nutanix/client.go

```diff
@@ -271,7 +626,6 @@ type ErrorResponse struct {
 
 // MessageResource ...
 type MessageResource struct {
-
```

Comment:
The type for `Details` was changed from `map[string]interface{}` to `interface` in https://github.com/nutanix-cloud-native/prism-go-client/commit/cc121d3d3c270c9d90419b8f9d0e19ef200f5702